src/python_testing/matter_testing_infrastructure/chip/testing/spec_parsing.py

```diff
@@ -150,9 +150,13 @@ class XmlDeviceType:
     # Keeping these as strings for now because the exact definitions are being discussed in DMTT
     classification_class: str
     classification_scope: str
+    superset_of_str: Optional[str] = None
+    superset_of: int = 0
```

Comment:
`_str` and without is not as obvious. Is this `superset_device_type_name` and `superset_device_type_id` ? Maybe that is clearer.